actionview/lib/action_view/template.rb

```diff
@@ -222,6 +227,18 @@ def encode!
       end
     end
 
+    # This method is responsible for marking a template as having explicit locals
+    # and extracting any arguments declared in the format
+    # locals: (message:, label: "My Message")
+    def explicit_locals!
+      self.source.sub!(EXPLICIT_LOCALS_REGEX, "")
+      @arguments = $1
+
+      return if @arguments.nil? # Magic comment not found
+
+      @locals_disabled = !@arguments.present?
```

Comment:
```suggestion
      @arguments = "**nil" if @arguments.blank?
```

This way you get rid of that extra instance variable.

Reply:
Oooh. This makes me wonder if we could get rid of @explicit_locals too 🤔

Reply:
I don't think you can as it's needed in a couple places to check the method signature, etc.